### ISSUE TITLE
Explain PHI-Canto's curation scope on the front page

### ISSUE THREAD

**Issue by jseager7** [pombase/canto#1693]
It might be useful to have some text on PHI-Canto's front page clarifying what sort of data PHI-base _doesn't_ collect: e.g. viruses; organisms not included in our 'top list'; etc.

@CuzickA I'll probably need help from you or someone else in the team to clarify what you don't expect to curate in the foreseeable future.

**Comment by CuzickA** [comment]
![image](https://user-images.githubusercontent.com/11753634/48833144-a54ad680-ed72-11e8-900f-54240a37cf20.png)

Thanks for the ticket @jseager7 
In our last meeting we decided to add the highlighted yellow text in the above mockup. We decided it was better to list the pathogens types that we ARE curating rather than those we are not (viruses). 

We also discussed whether to limit entries to top '200' pathogen species but decided not to do this. This means that we be able to capture data on new emerging pathogens.

**Comment by jseager7** [comment]
Thanks for this. I just have one suggestion: I think the text would be better placed with the rest of the information about the curation process under the PubMed ID field. Here's an example:

![valid_curation_notice](https://user-images.githubusercontent.com/37659591/48836888-d8de2e80-ed7b-11e8-973a-f5a9e38d7ea0.PNG)

(Note that the **emphasis is mine** in the image above; the yellow highlighting won't be there when displayed in PHI-Canto.)

I'm not sure if this text is currently configurable, so we might need a small change to the config file and the Perl loader to handle this.

**Comment by jseager7** [comment]
We've decided that the top of the page would be the best place for this notice. @CuzickA I propose replacing the text 'Add your work to PHI-base', with the text 'Currently curating fungal, protist, and bacterial pathogens'. Note that the new wording is subtly different, since I wanted it to seem less like a warning to the user. Here's a mockup:

![curation_notice](https://user-images.githubusercontent.com/37659591/49218376-27b54500-f3c8-11e8-976b-76a574328a52.PNG)

@kimrutherford Unfortunately, I've discovered that the top of the page isn't fully configurable either: currently only the database name is substitutable, and if we make the text fully configurable, then this will have knock-on effects for PomBase's Canto. The relevant code is in `instance_front.mhtml`:

https://github.com/pombase/canto/blob/fd256e448687479e074b3d3f5cbd501ece79465b/root/docs/instance_front.mhtml#L2

So I guess the option is either to make this text consist of nothing besides a configurable variable like `page_subtitle` &ndash; in which case PomBase's configuration would have 'Add your work to PomBase' &ndash; or to add another slot for the new text below the existing one.
